logger.js

```diff
@@ -0,0 +1,43 @@
+import winston from 'winston';
+winston.emitErrs = false;
+
+const logLevel = (process.env.NODE_ENV === 'production') ? 'info' : 'debug';
+
+const loggerTransports = [
+  new winston.transports.File({
+    level: logLevel,
+    filename: './log/discovery-ui.log',
+    handleExceptions: true,
+    maxsize: 5242880, // 5MB
+    maxFiles: 5,
+    colorize: false,
+    json: false,
+    formatter: (options) => {
+      const outputObject = {
+        level: options.level.toUpperCase(),
```

Comment:
You might want to connect with @rhernand3z as I think some of the Winston logging doesn't exactly follow our standard (https://github.com/NYPL/engineering-general/blob/master/standards/logging.md) and he's done some work on customizing the output.

See:
https://github.com/NYPL/nypl-hold-request-consumer/blob/master/src/helpers/Logger.js